students/15th/turtle/hanbyul-kim/user/views.py

```diff
@@ -0,0 +1,100 @@
+
+import json, re
+import bcrypt
+import jwt
+
+from django.http        import JsonResponse
+from django.views       import View
+from django.db          import IntegrityError
+from django.db.models   import Q
+
+from user.models        import User
+
+
+
+class UserView(View):
+    def post(self,request):
+        data = json.loads(request.body)
+
+        try:
+            data['name']    =   data.get('name')
+            data['email']   =   data.get('email')
+            data['mobile_number']   =   data.get('mobile_number')
+
+            if len(data['password']) < 8:
+                return JsonResponse({"message":"password must be at least 8 characters"},status=400)
+
+            email_reg='[a-zA-Z0-9_-]+@[a-z]+.[a-z]'
+            email_validation = re.compile(email_reg)
```

Comment:
* 한 줄로 줄일 수 있습니다.
* 상수는 대문자로 표현합니다.
```suggestion
            REGEX_EMAIL = re.compile('[a-zA-Z0-9_-]+@[a-z]+.[a-z]')
```